Source/WebCore/html/HTMLTableElement.cpp

```diff
@@ -81,9 +81,9 @@ ExceptionOr<void> HTMLTableElement::setCaption(RefPtr<HTMLTableCaptionElement>&&
 
 RefPtr<HTMLTableSectionElement> HTMLTableElement::tHead() const
 {
-    for (RefPtr<Node> child = firstChild(); child; child = child->nextSibling()) {
+    for (Ref child : childrenOfType<HTMLTableSectionElement>(const_cast<HTMLTableElement&>(*this))) {
```

Comment:
Seems like a `childrenOfType` bug that you need `const_cast` here.  Would be nice to fix it some day.